lib/gooddata/rest/connection.rb

```diff
@@ -479,6 +479,10 @@ def merge_cookies!(cookies)
         @request_params[:cookies].merge! cookies
       end
 
+      def process_error(error, params)
+        GoodData.logger.error(format_error(error, params))
+      end
+
       def process_response(options = {}, &block)
         retries = options[:tries] || 3
 
```

Comment:
Could you not commit this file? It is just a refactoring that is not pertinent to the story and I have exactly the same changes in the story for not logging token. It will clash anyway :-)